src/go/rpk/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/redpanda-data/redpanda/src/go/rpk
 
-go 1.20
+go 1.21
+
+toolchain go1.22.0
```

Comment:
There is a mismatch between these 2

Reply:
Meant to remove, it's now removed (squashed out)